.changeset/shiny-dingos-remember.md

```diff
@@ -0,0 +1,21 @@
+---
+"astro-remote": major
```

Comment:
I am questioning if a `major` is even required here! This is a bit of a catch-all PR, but updating dependencies can usually be handled as a `patch` and (at worst) a `minor`.

If there are specific commits that fix an issue, those should maybe be cherry picked onto a new branch and reviewed separately.